src/shared_utilities/Domains.jl

```diff
@@ -235,7 +235,7 @@ Example: `longlat = (FT(-118.14452), FT(34.14778))`. `longlat` is in degrees, wi
 is not accounted for. We compute `xlim` and `ylim` as
 
 ```julia
-xlim = (long - xlim[1] / (2radius_earth), long + xlim[2] / (2radius_earth))
+xlim = (long - xlim[1] / (2π*radius_earth) * 360, long + xlim[2] / (2π*radius_earth) * 360)
```

Comment:
There are functions `deg2rad` and `rad2deg`

Reply:
I might keep it this way for readability unless you feel strongly about it!

Reply:
I personally would find it more readable as
```
long - rad2deg(xlim[1] / 2radius_earth)
```
but I don't feel strongly about it, so have it as you prefer